tests/modbus/test.rules

```diff
@@ -1,3 +1,5 @@
+alert modbus any any -> any any (msg:"Testing modbus code function"; modbus: access write; sid:6;)
```

Comment:
This is maybe nit, but according to the documentation, this rule message should have a message like: `Testing modbus access` or something like that, I think...

cf https://suricata.readthedocs.io/en/latest/rules/modbus-keyword.html?highlight=modbus#modbus-keyword